mods/cnc/rules/defaults.yaml

```diff
@@ -813,6 +813,23 @@
 	RenderSprites:
 		Palette: terrain
 
+^CivHaystackOrIgloo:
+	Inherits: ^CivField
+	MapEditorData:
+		ExcludeTilesets: DESERT
+	-Tooltip:
+	GrantConditionOnTileSet@WINTER:
+		Condition: winter
+		TileSets: SNOW, WINTER
+	Tooltip@WINTER:
+		Name: Igloo
+		RequiresCondition: winter
+		GenericVisibility: Enemy, Neutral
+		ShowOwnerRow: false
```

Comment:
`GenericVisibility` and `ShowOwnerRow` values here seem to be identical to the campaign overrides, and the SUMMER tooltip below doesn't define overrides for them here, only in `campaign-tooltips.yaml`.
Is that intentional?